lib/luadata.c

```diff
@@ -34,6 +37,12 @@ static inline luadata_t *luadata_checkdata(lua_State *L, lua_Integer *offset, lu
 
 #define luadata_checkint(L, offset, T)	luadata_checkdata((L), &(offset), sizeof(T##_t))
 
+static inline void luadata_editablechecker(lua_State *L, luadata_t *data)
+{
+	if (data->flags & LUADATA_CONST)
+		luaL_error(L, "data is not editable");
```

Comment:
```suggestion
		luaL_error(L, "not editable");
```
I would follow the pattern of the other error messages.. e.g., "out of bounds".. perhaps "read only" would be a better message in this case.. I'm not very fond of `editable`, btw